env_files/back-safe.env

```diff
@@ -5,7 +5,7 @@ POSTGRES_HOST=postgres
 NLP_HOST=cl2-nlp:8000
 RABBITMQ_URI=amqp://guest:guest@rabbitmq:5672
 SEED_SIZE=small
-OVERRIDE_HOST=localhost
+OVERRIDE_HOST=product.stg.citizenlab.co
```

Comment:
Not sure you meant to check this in?

Reply:
That was accidentally committed along with the other files 😅 Thanks for spotting!